utils/readsb/Makefile

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2020 Stijn Tintel <stijn@linux-ipv6.be>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=readsb
+PKG_VERSION:=3.8.3
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://github.com/Mictronics/readsb/archive/v$(PKG_VERSION)
```

Comment:
I think codeload should be used here.

Reply:
My package vallumd doesn't use this either, so I prefer to keep them consistent. If using codeload is *really* a must, we should update everything that doesn't use it, and then auto-reject PKG_SOURCE_URL starting with https://github.com/ via CI with a friendly message to use codeload, and how/why.

Reply:
Eh no.

The typical pattern with github is to use codeload for packages that have proper versioning and local tarballs for packages that do not.

Note that that URL redirects to codeload anyway. I think redirects are to be avoided here.

Reply:
Changed to codeload.